packages/target-tauri/src-tauri/capabilities/main-window.toml

```
@@ -19,6 +19,9 @@ permissions = [
     "store:allow-save",
     "store:allow-set",
     "store:allow-delete",
+    # events

```

let's say it is only for drag and drop events, because we explicitly don't use events for other things anymore. we migrated to using channels for security (events can be seen by all windows).

```suggestion
    # events - only used for drag and drop
```